docs/en/08_troubleshooting.md

```diff
@@ -95,3 +95,127 @@ _-: php: command not found
 This restriction is a security feature coming with Plesk 10.
 On round about page 150 of the plesk Administrator Guide you will find a solution to enable scheduled tasks which use the command line. (But the latest Guide for 10.3.1 mentions `/usr/local/psa/admin/bin/server_pref -u -crontab-secure-shell "/bin/sh"` although "server_pref" doesnt exit.
 Since we are using a dedicated server for only one customer, we defined the crons under "Server Management"->"Tools & Utilities"->"Scheduled Tasks"->"root". The security restrictions of plesk are not involved then.
+
+## Stuck jobs
+
+Sometimes, jobs break without having any issues with their implementation but rather an external factor is the root cause, for example database table lock may prevent a DB write.
+Most common scenario is "publish" action related jobs such as scheduled publish feature which may experience DB deadlocks on the versioned table as this can be frequently accessed.
+In other cases, a job can get paused by a queue runner due to lack of server resources at a particular time.
+Paused jobs can usually be safely resumed, continuing from the last completed step at a later time.
+
+For these scenarios it's recommended to configure automatic job retries.
+Configuration has some flexibility on number of retries and the timing of the retry attempts.
+
+### Basic configuration
+
+This configuration is recommended as a good starting point when trying to set up automatic retries.
+
+- `max_retry_attempts` - number of retry attempts, this allows to control how many times a stuck job is retried
+- `initial_retry_delay` - minimal waiting time in seconds before the retry attempt is executed, this helps spread retry attempts apart from each other
```

Comment:
```suggestion
- [`AbstractQueuedJob.max_retry_attempts`](api:Symbiote\QueuedJobs\Services\AbstractQueuedJob->max_retry_attempts) - number of retry attempts, this allows to control how many times a stuck job is retried
- [`AbstractQueuedJob.initial_retry_delay`](api:Symbiote\QueuedJobs\Services\AbstractQueuedJob->initial_retry_delay) - minimal waiting time in seconds before the retry attempt is executed, this helps spread retry attempts apart from each other
```

The link won't work in GitHub but links to the API docs when viewed in https://docs.silverstripe.org/en/optional_features/queuedjobs/troubleshooting/

Prefixing the configuration property with the name of the class that defines it is standard practice for Silverstripe CMS docs and gives an immediate clue as to its usage.